vhost/src/vhost_user/message.rs

```diff
@@ -990,6 +994,48 @@ impl VhostUserMsgValidator for VhostUserTransferDeviceState {
     }
 }
 
+// Bit mask for flags in VhostUserBackendMapMsg backend message
+bitflags! {
+    #[derive(Copy, Clone, Debug, Eq, PartialEq, Default)]
+    /// Flags for virtio-fs backend messages.
```

Comment:
virtio-fs?

Reply:
Oops, fixed it. It was a leftover from the removed virtio-fs DAX messages.